### ISSUE TITLE
Option to decrease?

### ISSUE THREAD

**Issue by K0-RR** [dvitski/undergroundambientlighting#3]
I want my caves pitch black as they should be, any chance you will add this as a config?

**Comment by dvitski** [comment]
You can press the `I` keybind (changeable in the controls menu) to disable the mod temporarily

**Comment by K0-RR** [comment]
But... I want them darker than vanilla.

**Comment by dvitski** [comment]
Oh, umm
Brightness at moody might get that effect, I guess?

Interesting suggestion though

**Comment by mr-pi12** [comment]
>But... I want them darker than vanilla.

The boosted brightness mod allows you to set negative brightness levels, you could try using it to achieve the desired effect?

**Comment by Xrey274** [comment]
Yeah adding a config menu with a modifier value, would be awesome. I personally would like the decrease the brightness of the mod (whist still being brigther than vanilla) just a bit.
